Translation/Extractor/File/DefaultPhpFileExtractor.php

```diff
@@ -159,19 +159,36 @@ public function enterNode(Node $node)
         }
 
         $id = $node->args[0]->value->value;
-
         $index = $this->methodsToExtractFrom[strtolower($methodCallNodeName)];
-        if (isset($node->args[$index])) {
-            if ($node->args[$index]->value instanceof Node\Expr\ConstFetch && 'null' === (string) $node->args[$index]->value->name) {
+        $domainArg = null;
+
+        if (isset($node->args[$index]) && $node->args[$index] instanceof Node\Arg && null === $node->args[$index]->name ) {
```

Comment:
`name` property for Argument was added in PHP-Parser 4.9.0.
Maybe it's judicious to bump the PHP-Parser version to `^4.9` in composer.json?

Reply:
that would be a good idea, can you add it please?

Reply:
@goetas just did it, sorry for the delay.

Reply:
@goetas would you have some time to have another look now I updated the dependencies.